pages/common/git-mailinfo.md

```diff
@@ -0,0 +1,16 @@
+# git mailinfo
+
+> Extract patch and authorship information from a single email message.
+> More information: <https://git-scm.com/docs/git-mailinfo>.
+
+- Extract the patch and author data from an email message:
+
+`git mailinfo {{message|patch}}`
+
+- Extract but remove leading and trailing whitespace:
```

Comment:
```suggestion
- Extract but remove leading and trailing whitespaces:
```

Reply:
This isn't really correct — whitespace is plural and singular.

Reply:
Oh, good to know.